src/settings-ui/Settings.UI/Views/VideoConference.xaml

```diff
@@ -117,15 +120,15 @@
                                     <Button
                                         x:Uid="VideoConference_CameraOverlayImageClear"
                                         Command="{Binding Mode=OneWay, Path=ClearOverlayImage}"
-                                        Visibility="{Binding Path=CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToCollapsedConverter}}" />
+                                        IsEnabled="{Binding Path=CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToBoolConverter}}" />
                                 </StackPanel>
                             </labs:SettingsCard>
-                            <labs:SettingsCard>
+                            <labs:SettingsCard
+                                Visibility="{Binding Path=CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToCollapsedConverter}}">
```

Comment:
@niels9001 When I change **EmptyToCollapsedConverter** to **StringVisibilityConverter**, things break. Do you know why?